src/VecSim/algorithms/brute_force/brute_force.h

```diff
@@ -303,14 +308,14 @@ BruteForceIndex<DataType, DistType>::rangeQuery(const void *queryBlob, double ra
         }
```

Comment:
On range queries, on timeout we want to return whatever results we got because we know they are valid. You can use `break` or `goto` here, or just add the `get_results` call